lib/src/legacy/utils.ts

```diff
@@ -38,7 +38,13 @@ export function removeLegacyImporter(string: string): string {
 // syntax.
 export function removeLegacyImporterFromSpan(span: SourceSpan): SourceSpan {
   if (!span.url) return span;
-  return {...span, url: new URL(removeLegacyImporter(span.url.toString()))};
+  const url = removeLegacyImporter(span.url.toString());
+  try {
+    return {...span, url: new URL(url)};
+  } catch (_) {
+    // Relative URL
+    return {...span, url: new URL(url, `file://${process.cwd()}`)};
```

Comment:
Just shoving a pathname into a URL string isn't safe in general. Use Node's [`pathToFileUrl()`](https://nodejs.org/api/url.html#urlpathtofileurlpath-options).